tokenserver-db-sqlite/src/models.rs

```diff
@@ -0,0 +1,111 @@
+pub const LAST_INSERT_ID_QUERY: &'static str = "SELECT LAST_INSERT_ROWID() AS id";
```

Comment:
Same as with the mysql file, it'd be nice to add a bit of space here so they're easier to scan.